contracts/scripts/Deploy.sol

```diff
@@ -1,38 +1,83 @@
 // SPDX-License-Identifier: MIT
 pragma solidity ^0.8.16;
 
-import 'forge-std/Test.sol';
+import "forge-std/Test.sol";
 
-import '@jbx-protocol/juice-contracts-v3/contracts/interfaces/IJBOperatorStore.sol';
-import '@jbx-protocol/juice-contracts-v3/contracts/interfaces/IJBProjects.sol';
+import "@jbx-protocol/juice-contracts-v3/contracts/interfaces/IJBOperatorStore.sol";
+import "@jbx-protocol/juice-contracts-v3/contracts/interfaces/IJBProjects.sol";
 
-import '../JBProjectHandles.sol';
+import "../JBProjectHandles.sol";
 
-contract DeployGoerli is Test {
-  IJBOperatorStore _operatorStore = IJBOperatorStore(0x99dB6b517683237dE9C494bbd17861f3608F3585);
-  IJBProjects _projects = IJBProjects(0x21263a042aFE4bAE34F08Bb318056C181bD96D3b);
-  IJBProjectHandles _oldHandle = IJBProjectHandles(0x41126eC99F8A989fEB503ac7bB4c5e5D40E06FA4);
+contract DeploySepolia is Test {
+    IJBOperatorStore _operatorStore = IJBOperatorStore(0x8f63C744C0280Ef4b32AF1F821c65E0fd4150ab3);
+    IJBProjects _projects = IJBProjects(0x43CB8FCe4F0d61579044342A5d5A027aB7aE4D63);
+
+    // Have to use its own address as the `_oldHandle`.
+    // TODO: UPDATE THIS EACH TIME YOU DEPLOY.
+    address willBeDeployedAt = _addressFrom(0xfAb4c9E48EB050E8f51555Ef949a7ef1b8fb263B, 0);
```

Comment:
what's this address? It should be the deployer, so msg.sender *should* be good (as for the nonce, use `vm.getNonce(deployer address)`?)

Reply:
I used this suggestion like so:

```solidity
    address willBeDeployedAt = _addressFrom(msg.sender, vm.getNonce(msg.sender));
```

And it didn't work when I tried it locally on an anvil fork of sepolia. I also tried using nonce + 1 – also did not work.

If you can get it working, please do! Otherwise I think it's fine to just update the address on new deployments.